src/include/migraphx/op/unary.hpp

```diff
@@ -21,7 +21,7 @@ struct unary
     shape compute_shape(std::vector<shape> inputs) const
     {
         check_shapes{inputs}.has(1);
-        return inputs.at(0);
+        return {inputs.at(0).type(), inputs.at(0).lens()};
```

Comment:
I wonder why this is needed?

Reply:
The unary operator can accept input of non-standard (like transposed) shape and its output shape is supposed to be standard since the unary internally will handle the nonstandard input. If the input shape is e.g., transposed, the current implement returns a transposed shape, which is incorrect.

Reply:
perhaps add a comment to reference this in the future?